src/test/resources/s3proxy.conf

```diff
@@ -1,6 +1,6 @@
-s3proxy.endpoint=https://127.0.0.1:0
+s3proxy.endpoint=http://127.0.0.1:0
 # authorization must be aws-v2 or none
-s3proxy.authorization=aws-v2
+s3proxy.authorization=AWS_V2
```

Comment:
why change this?

Reply:
I was able to easily convert it to `AuthenticationType`:

```
AuthenticationType.valueOf(s3ProxyProperties.getProperty(S3ProxyConstants.PROPERTY_AUTHORIZATION));
```

Reply:
I wonder why this code doesn't call `S3Proxy.Builder.fromProperties`?

Reply:
Cool! I'll try to refactor it.

Reply:
This is really messy. In order to keep this PR clean and simple, I suggest to do tests refactoring separately. In this PR I'll write it a way so that `s3proxy.authorization=aws-v2` remains unchanged.